addons/ui_units/initSettings.sqf

```diff
@@ -0,0 +1,47 @@
+[
+    QGVAR(landSpeed),
+    "LIST",
+    [localize LSTRING(Speed_DisplayName), localize LSTRING(Speed_Description)],
+    [format["ACE %1", localize LSTRING(Module_DisplayName)], localize LSTRING(Land_DisplayName)],
+    [[SPEED_KMH, SPEED_MPH, SPEED_KNOT, SPEED_MS], [LSTRING(Speed_KMH), LSTRING(Speed_MPH), LSTRING(Speed_KNOT), LSTRING(Speed_MS)], 0]
+] call CBA_settings_fnc_init;
```

Comment:
Is it really a good idea to have those local? What units a car shows you doesn't depend on the driver, but on the car. Additionally, I don't want to impose confusion when people run different units, so making those global settings would be ideal.

Reply:
If you don't want to confuse people with different settings on the clients, force one or the other. But leave the option to not force this settings for communities that don't care.